app/parser/v0_0_1/schema_parser.py

```diff
@@ -251,7 +248,7 @@ def _parse_question(self, schema, questionnaire):
             question.id = ParserUtils.get_required_string(schema, "id")
             question.title = ParserUtils.get_required_string(schema, "title")
             question.description = ParserUtils.get_required_string(schema, "description")
-            question.skip_condition = self._parse_skip_condition(ParserUtils.get_optional(schema, "skip_condition"))
+            question.skip_condition = schema.get("skip_condition")
```

Comment:
I'd agree it's against the grain of how the schema is now but knowing that we are going to be removing the schema model imminently to use a dict anyway is it worth it also it's only being used in that one place?